shapeshift/web
# feat: bump hdwallet to latest

## Description



Included changes: 
- https://github.com/shapeshift/hdwallet/pull/616
- https://github.com/shapeshift/hdwallet/pull/614

## Pull Request Type


## Issue (if applicable)



- closes https://github.com/shapeshift/web/issues/4321

## Risk



## Testing



- Ensure MetaMask Txs use EIP-1559 fees scheme for signing, vs. legacy
  currently
- Ensure BCH sends on KK are working

### Engineering



- :point_up:

### Operations



- :point_up:

## Screenshots (if applicable)
